pytx3/pytx3/cli/fetch2.py

```diff
@@ -59,16 +65,72 @@ def __init__(
         self.owner = owner
         self.threat_types = threat_types
         self.additional_tags = additional_tags
+        self.signal_types_by_name = {
+            name: signal()
+            for name, signal in meta.get_signal_types_by_name().items()
+        }
+        self.last_update_printed = 0
```

Comment:
why not start at `time.time()` instead of 0?

Reply:
Alteratively, we could do something similar to `fetch.py` that uses

```python
# Cause first print to be after ~5 seconds
self.last_update_printed = now - self.PROGRESS_PRINT_INTERVAL_SEC + 5
```

Reply:
Used 0 as an initial value so we would print right after the first fetch so the user gets some immediate feedback (so they know the command is doing stuff) vs having to wait 30s before the feedback.

Could do something similar to fetch.py if you prefer but I don't think it makes much of a difference

Reply:
Either way is fine with me. We can iterate on the UX for the command too once it is out of experimental.